pkgs/development/libraries/cjson/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchFromGitHub, cmake}:
+
+stdenv.mkDerivation rec {
+  pname = "cjson";
+  version = "1.7.14";
+
+  src = fetchFromGitHub {
+    owner = "DaveGamble";
+    repo = "cJSON";
+    rev = "v${version}";
+    sha256 = "1a3i9ydl65dgwgmlg79n5q8qilmjkaakq56sam1w25zcrd8jy11q";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/DaveGamble/cJSON";
+    description = "Ultralightweight JSON parser in ANSI C ";
+    license = licenses.mit;
```

Comment:
A part of the test suite is licensed under [ASL 2.0](https://github.com/DaveGamble/cJSON/blob/master/tests/json-patch-tests/README.md#license).

Reply:
How to have multiple licenses in a package in nixpkgs?

Reply:
```
license = with licenses; [ mit asl20 ];
```